From 8f76cc9d5f0b50e491f3c00724546050fc4eb376 Mon Sep 17 00:00:00 2001
From: Changming Huang <jerry.huang@nxp.com>
Date: Wed, 17 Nov 2021 18:49:18 +0800
Subject: [PATCH 18/22] ls1028ardb: remove gic distributor region from
 linux-enetc demo

when this region is enabled in inmate cell,
there are some commands (related to hardware) can't be implemented.

Signed-off-by: Changming Huang <jerry.huang@nxp.com>
Upstream-Status: Pending
---
 configs/arm64/ls1028a-rdb-linux-enetc-demo.c | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/configs/arm64/ls1028a-rdb-linux-enetc-demo.c b/configs/arm64/ls1028a-rdb-linux-enetc-demo.c
index 5220a91d..97914af6 100644
--- a/configs/arm64/ls1028a-rdb-linux-enetc-demo.c
+++ b/configs/arm64/ls1028a-rdb-linux-enetc-demo.c
@@ -16,7 +16,7 @@
 struct {
 	struct jailhouse_cell_desc cell;
 	__u64 cpus[1];
-	struct jailhouse_memory mem_regions[12];
+	struct jailhouse_memory mem_regions[11];
 	struct jailhouse_irqchip irqchips[2];
 } __attribute__((packed)) config = {
 	.cell = {
@@ -65,14 +65,6 @@ struct {
 				JAILHOUSE_MEM_IO | JAILHOUSE_MEM_ROOTSHARED,
 		},
 
-		/* gic distributor */ {
-			.phys_start = 0x06000000,
-			.virt_start = 0x06000000,
-			.size = 0x10000,
-			.flags = JAILHOUSE_MEM_READ | JAILHOUSE_MEM_WRITE |
-				JAILHOUSE_MEM_IO | JAILHOUSE_MEM_ROOTSHARED,
-		},
-
 		/* gic re-distribuitor */ {
 			.phys_start = 0x06060000,
 			.virt_start = 0x06060000,
-- 
2.43.0

